### PR TITLE
Dane FAQ 2.0 Feb 2022

### PR DESCRIPTION
**Pull request by bradleysigma**
Changes for the recently updated Dane FAQ
https://docs.google.com/document/d/1_93kw7KkzvAXoQAxRioxo7iAt-Ar6BMIxdNf4Lv25rI/edit

### CODE REVIEW

File: F_ul.php [hunk 1]
```diff
@@ -8,7 +8,6 @@
 <h1>Awaken <sub></sub></h1>
     <ol class="note">
     <li>Only sleeper tokens present in the system when it is activated may be replaced with PDS.</li>
-    <li>The Ul player cannot place a PDS on a planet that already has two of their PDS on it. As such, they are unable to place a Hecatoncheires on a planet with two of their PDS using its <sc>Deploy</sc> ability.</li>
```
Comment (ctinfty) [inline F_ul.php:11]: Why is this removed? The pertinent question is IMO

> Q: Does 63.3 (2 PDS on a planet max) mean that Titans couldn't place a PDS via Sleeper token onto a planet where another player already has 2 PDS? Related, if a planet has 2 PDS on it already can you DEPLOY the Titan’s mech there, given that the mech is "when you would place a PDS"?
A: 63.3 does not prevent Titans from placing a PDS on a planet that contains 2 of another player’s PDS units. You cannot place a structure on a planet with the maximum number of your structures, which **prevents using the DEPLOY ability on a planet with 2 PDS of yours**, or placing more PDS than would be allowed using Transit Diodes. You also cannot DEPLOY a mech if you have no PDS in reinforcements, but could remove a PDS from the board and then DEPLOY your mech to an eligible target planet

and that has not changed.

File: F_xxcha.php [hunk 2]
```diff
@@ -57,19 +57,23 @@
...
     <li>If the Xxcha player draws, reveals and resolves the <i>Miscount Disclosed</i> agenda, they cast one vote for an outcome of the elected law, and resolve it.</li>
     <li>If the Xxcha player draws, reveals and resolves the <i>Public Execution</i> agenda, the elected player may still vote in the agenda phase following the current action phase.</li>
     <li>When the Xxcha player chooses and reveals an agenda, they may then use their <sc>Quash</sc> ability.</li>
     <li>After the Xxcha player chooses and reveals an agenda, they may play any rider action cards on the revealed agenda.</li>
     <li>The Xxcha player chooses the order the two agendas are resolved.</li>
     <li>The two agendas are revealed before either is resolved.</li>
     <ol><li>The Xxcha player will resovle all abilities that trigger when or after an agenda is revealed for the first agenda before revealing the second agenda.</li></ol>
+    <li>When an agenda outcome with an immediate effect resolves, other players resolve the effect if it is mandatory, but cannot resolve resolve it if it is optional.</li>
```
Comment (ctinfty) [inline F_xxcha.php:68]: I assume this is consensus from the Discord Rules Experts as I cannot infer this from the FAQ directly.

File: F_ul.php (hunk 1 above)
Comment (bradleysigma) [reply F_ul.php:11]: This note also appears in the "Hecatoncheires (Mech)" section. It seemed redundant to duplicate it, and it's more directly relevant to the DEPLOY ability of the mech.

File: F_xxcha.php (hunk 2 above)
Comment (bradleysigma) [reply F_xxcha.php:68]: Yes, this is from the Discord.

File: F_nekro.php [hunk 3]
```diff
@@ -35,6 +35,7 @@
 <h1>Antivirus <sub>(Promissory Note)</sub></h1>
     <ol class="note">
     <li>The effect of <i>Antivirus</i> does not prevent the Nekro player gaining a technology from a player via their <sc>Galactic Threat</sc> ability.</li>
+    <li>The player holding <i>Antivirus</i> may only play it at the start of a combat between them and the Nekro player.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_nekro.php:38]: Whoops, you are correct.

**Pull request merged**